libnavigation-base/src/main/java/com/mapbox/navigation/base/extensions/RouteRefreshEx.kt

```diff
@@ -1,15 +1,27 @@
-package com.mapbox.navigation.base.internal.extensions
+@file:JvmName("DirectionsRefreshEx")
```

Comment:
`DirectionsRefresh` is not exist, it's an abstract name. I would rename  
```suggestion
@file:JvmName("RouteOptionsEx")
```